composer.json

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
It is needed for using pure PHP runtime. Basically it describes the PHP runtime package like Protobuf.spec for objc. It has to be on the top level to be correctly pulled by the central PHP repository.